.github/workflows/terraform-lint.yaml

```diff
@@ -21,27 +21,58 @@ jobs:
 
       - name: terraform fmt
         run: terraform fmt -no-color -check -diff -recursive
-  tfsec:
-      name: tfsec
+  tflint:
+      name: TFLint
+      runs-on: ubuntu-latest
+
+      permissions:
+        contents: read
+
+      steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Cache plugin dir
+        uses: actions/cache@v3
+        with:
+          path: ~/.tflint.d/plugins
+          key: tflint-${{ hashFiles('.tflint.hcl') }}
+
+      - name: Setup TFLint
+        uses: terraform-linters/setup-tflint@v3
+
+      - name: Show version
+        run: tflint --version
+
+      - name: Init TFLint
+        run: tflint --init
+
+      - name: Run TFLint
+        run: tflint -f compact
+  trivy:
+      name: trivy
       runs-on: ubuntu-latest
 
       permissions:
-        actions: read
         contents: read
-        pull-requests: write
         security-events: write
 
       steps:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: tfsec
-        uses: aquasecurity/tfsec-pr-commenter-action@v1.3.1
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
```

Comment:
Lets pin this at a known version, and upgrade it as we do for our other dependencies.

Reply:
Updated with a pin to their latest 0.9.1